diff --git a/Makefile.am b/Makefile.am
index cbbe124..bf8c9cd 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,5 @@
 bin_SCRIPTS = xmlto
 bin_PROGRAMS = xmlif/xmlif
-dist_man1_MANS = man/man1/xmlto.1 man/man1/xmlif.1
 
 xmlif_xmlif_SOURCES = xmlif/xmlif.l
 
@@ -68,15 +67,10 @@ EXTRA_DIST = xmlto.spec \
 	doc/xmlif.xml \
 	xmlto.mak
 
-GEN_MANPAGE = FORMAT_DIR=$(top_srcdir)/format $(XMLTO_BASH_PATH) ./xmlto --skip-validation -o $(@D) man $<
-man/man1/xmlto.1: doc/xmlto.xml ; $(GEN_MANPAGE)
-man/man1/xmlif.1: doc/xmlif.xml ; $(GEN_MANPAGE)
-
 TESTS_ENVIRONMENT = top_srcdir=$(top_srcdir) top_builddir=$(top_builddir)
 TESTS = xmlif/test/run-test
 
-CLEANFILES=$(dist_man1_MANS) \
-	xmlif/test/test-no-condition \
+CLEANFILES=xmlif/test/test-no-condition \
 	xmlif/test/test-unrelated-condition \
 	xmlif/test/test-html \
 	xmlif/test/test-pdf \

